Roni reward ads

ronigrr opened the pull request.

ronigrr committed:
endgame dialog to a separated class

ronigrr committed:
endgame dialog to a separated class

ronigrr committed:
endgame dialog to a separated class
test rewards (+10 hints in test)

ronigrr committed:
endgame dialog to a separated class
test rewards (+10 hints in test)

ronigrr committed:
endgame dialog to a separated class
test rewards (+10 hints in test)

ronigrr committed:
endgame dialog to a separated class
test rewards (+10 hints in test)
minor code fix

ronigrr committed:
endgame dialog to a separated class
test rewards (+10 hints in test)
minor code fix
intro gif!

ronigrr committed:
endgame dialog to a separated class
test rewards (+10 hints in test)
minor code fix
intro gif!

ilya3a commented on app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt:
```diff
@@ -410,7 +420,7 @@ class FragmentArcadeModeScreen : Fragment(), View.OnClickListener, IFinishTimer
             if (isDivideZero || isFraction) {
                 AudioManager.startBuzzerSound()
-                showFinishDialog(Constants.LOSE_DIALOG)
+                //showFinishDialog(Constants.LOSE_DIALOG)
```
why to comment??

ronigrr replied:
removed

ilya3a commented on app/src/main/java/com/example/makeiteven2/fragments/FragmentArcadeModeScreen.kt:
```diff
@@ -445,7 +455,7 @@ class FragmentArcadeModeScreen : Fragment(), View.OnClickListener, IFinishTimer
                     //you loose
                     Handler().postDelayed({
-                        showFinishDialog(Constants.LOSE_DIALOG)
+                        //showFinishDialog(Constants.LOSE_DIALOG)
```
also?

ronigrr replied:
I'm not sure if to show dialog here or not, see after merge it's in your fragments responsibility

ilya3a commented on app/src/main/java/com/example/makeiteven2/fragments/FragmentLevelsScreen.kt:
```diff
@@ -12,31 +12,34 @@ import com.example.makeiteven2.R
         return rootView
     }
 
+    private fun initLevelsAdapter() {
```
????? empty fun?

ronigrr replied:
oo come on I just forgot to remove!

ilya3a commented on app/src/main/java/com/example/makeiteven2/intefaces/IEndDialogBtnClicked.kt:
```diff
@@ -0,0 +1,7 @@
+
+import android.view.View
+
+interface IEndDialogBtnClicked {
```
change the name to Listener in the end - more readable

ronigrr replied:
done

ronigrr committed:
endgame dialog to a separated class
test rewards (+10 hints in test)
minor code fix
intro gif!
fixed change requests
